backend/src/openbeheer/config/api/views.py

```diff
@@ -25,10 +25,24 @@ class OIDCInfoView(APIView):
         },
     )
     def get(self, request: Request, *args, **kwargs):
-        config = OIDCClient.objects.get(identifier=OIDC_ADMIN_CONFIG_IDENTIFIER)
+        client = OIDCClient.objects.filter(
+            identifier=OIDC_ADMIN_CONFIG_IDENTIFIER
+        ).first()
+
+        # FIXME: I think that pytest reverts the transactions between tests and the data created
+        # in the post migrate signals is also deleted. So then there are no OIDCClients.
+        # This is a workaround...
```

Comment:
Yes. Looks like we can reload expected data in a `"function"` scope `django_db_setup` fixture that itself depends on the default one
https://pytest-django.readthedocs.io/en/latest/database.html#populate-the-test-database-if-you-use-transactional-or-live-server